geniza/corpus/templates/corpus/snippets/document_option_label.html

```diff
@@ -0,0 +1,25 @@
+{# template snippet for displaying document label on a form #}
+{# used on document merge form to provide enough information to merge accurately #}
+<div class="merge-document-label">
+    <strong><a target="_blank" href="{% url 'admin:corpus_document_change' document.id %}">{{ document }}</a></strong>
+    <p>PGPID {{ document.id }}</p>
+    {% if document.description %}
+        <p>
+            <dt>Description</dt>
+            <dd>{{ document.description }}</dd>
+        </p>
+    {% endif %}
+
+    {% if document.footnotes.count %}<strong>Footnotes</strong>{% endif %}
+    <ul>
+        {% for footnote in document.footnotes.all %}
+            <li><a target="_blank" href="{% url 'admin:footnotes_footnote_change' footnote.id %}">{{ footnote }}</a>
+                {% if footnote.source %}<dt>Source</dt>
+                    <dd>{{ footnote.source }}</dd>{% endif %}
+                {% if footnote.url %}<dt>URL</dt>
+                    <dd><a target="_blank" href="{{ footnote.url }}">{{ footnote.url }}</a></dd>{% endif %}
+            </li>
+
```

Comment:
This display is not very usable. So that we don't have to reinvent the wheel here, can we adapt existing designs and use the pubic view format in `document_scholarship.html`, and add in the pencil icon with links to the admin change form?  (I don't know if there may be display problems because we don't have the styles here — I don't propose we add them, it doesn't need to look the same — just suggesting we should adapt that display if possible.)

Reply:
Rachel said that it's really important for them to know which footnotes have content attached; you're showing the links (although not the labels), but you're not showing any transcription content. 

If you look at the admin footnote change form, you'll see how we're displaying it there — I think you should include the same template snippet here (`footnotes/transcription.html`). However, because transcriptions can be very long, I suggest wrapping the transcription in a disclosure element — otherwise I think it may be hard for them to view the documents together and compare.